build_tools/bazel/iree.bazelrc

```diff
@@ -277,7 +273,6 @@ build:remote_cache_bazel_ci --config=_remote_cache_base
 # included for clarity and so that this reference is automatically updated by
 # manage_images.py
 build:remote_cache_bazel_ci --host_platform_remote_properties_override='properties:{name:"cache-silo-key" value:"gcr.io/iree-oss/swiftshader-bleeding-edge@sha256:b3cce73eb9f41d67981bc8f00e98fd66fe3487caec1bcbf38d4039dcc61e499d"}'
-build:remote_cache_bazel_tf_ci --host_platform_remote_properties_override='properties:{name:"cache-silo-key" value:"gcr.io/iree-oss/frontends-swiftshader@sha256:da14cc93637d3bfad469a670d4d7a49982df5d107b775331965e3bacb981d4cf"}'
```

Comment:
@GMNGeoffrey I think this was only used by a Buildkite pipeline that is no longer in use. Seemed safe to remove. WDYT?

Reply:
I still see [usages](https://github.com/search?q=remote_cache_bazel_tf_ci+repo%3Aopenxla%2Firee&type=code) in the repo. Those either need to be updated or deleted (and then usages of them deleted, etc.). @pzread would know best about the current status of buildkite benchmark pipelines and whether the same scripts are being used for running locally as well.

Reply:
Oh, I see you're doing that with https://github.com/openxla/iree/pull/13435. Then yeah this LGTM assuming that is submitted first

Reply:
Yep. I can shuffle code between PRs as needed (some of the changes in that other PR may be more controversial).